friends/src/components/Friend.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const DeleteBtn = styled.button `
+    border-radius: 50%;
+    background-color: crimson;
+    margin-bottom: 10px;
+    position: relative;
+    bottom: 85px;
+    right: 50px;
+`;
+
+const FriendCont = styled.div `
+    background-image: linear-gradient(to right, pink, lightyellow, lightblue);
+    border: 3px solid black;
+    border-radius: 50%;
+    margin: 10px auto;
+    width: 45%;
+`;
+
+const DetailSpan = styled.span `
+    font-weight: bold;
+`;
+
+const FriendTitle = styled.h1 `
+    text-decoration: underline;
+`;
+
+const FriendDetails = styled.div `
+    /* border: 1px solid black; */
+`;
+
+const FriendP = styled.p `
+    font-size: 18px;
+`;
+
+class Friend extends React.Component {
+    constructor(props){
+        super(props);
+
+    }
+
+    render() {
+        return(
+            <FriendCont>
+                <FriendDetails className = 'friendInfo'>
+                    <FriendTitle>{this.props.friend.name}</FriendTitle>
```

Comment:
this looks like a good spot for object deconstruction
```
const {name, age, email} = this.props.friend
```
```suggestion
                    <FriendTitle>{name}</FriendTitle>
```